scripts/repocheck.py

```diff
@@ -97,8 +97,7 @@
   'contact',
   'contact-email',
   'url',
-  'license',
-  'description',
```

Comment:
@lnielsen Should description be checked for here? The only way to pass this test is to copy-paste the description from the setup.py into the CC boostrap (at which point this tests is not catching anything meaningful?).